draft-liu-nasr-architecture.md

```diff
@@ -32,18 +32,105 @@ author:
+  Path     │                                                               │  Path        
+  Evidence │                                                               │  Evidence    
+  (PE)     │                                                               │  (PE)        
+      ┌────▼───────────────┐       ┌────────────────────┐       ┌──────────┴─────────┐    
+      │                    │       │                    │       │                    │    
+      │      Attester      ├───────►     Attester...    ├───────►      Attester      │    
+      │                    │       │                    │       │                    │    
+      └────────────────────┘       └────────────────────┘       └────────────────────┘    
+                        Update PE with                Update PE with                      
+                          AR/RE/PoT                     AR/RE/PoT                         
+
+[NASR Architecture](https://docs.google.com/presentation/d/1sT2jn8YfgDE7KXhTuAgPBdlWDbxV2uZ_Z78SkMs0HNk/edit?usp=sharing)
+
+[ASCII Diagram](https://asciiflow.com/#/share/eJzVlb1OwzAQx1%2FF8tRKVSoxAO3WIStE5mPKYrWGRgpJcV1oVFVCFSMDQwR9DkbE0%2BRJcNIo35%2B1g8TJgyNffnfJ3f29gRZ%2BIHAMLyZXCA6giR1C%2BeNGh2sdjkeno4EOHb47OT%2FjO0bWjD%2FoEHDz3B%2FBBURM160wi6JTuXRETMew7oGGKXOk07P2X%2Bgt1sdXF9XXMJtnvkiUm6Aj8rgiS5alI7KwKROmJ4hFe2F6rgKdTKfnvnnui8B6B037NSj2hDFeEswM2%2BKFWK5MBnraBPV9t13x27tchEs6nXMKxcymFYGLUdxuCTXuDELLI%2BRe%2FXwVV0vh4jXKtOW%2FqI7wB%2FeD57o1PeZrT4XDHpRP5HGWJXB6qFTS6BEvoKtPxoxYUyKLHvECek9T%2B%2FGZOD3iJTolrSK%2BVrVWkZAvp%2BUa6Fp1jildq5iViv%2BazKbJtIU%2BB4mM1ansRv5OeSuK0sQ9Ry%2FKoOPvrJv3mrWPw8rqFQkZJRUoZTeLGWYEaCp4NqKZb3gaWCmZFxMNkTrU7OsjTn0y3MLtL2JpUd8%3D))
+
+
+# Roles {#roles}
+
+The existing roles from RATS Architecture document {{RFC9344}} applies. New role(s) are defined below.
+
+Orchestrator: A role performed by an entity (typically a controller or a special device) that receives a Path Request from the Relying Party, send unfilled Path Evidence (PE) inquiry to Attesters, collects Path Attestation Result (PAR) from the Verifier, and send PAR back to the Relying Party.
+
+    Consumes: Path Request, Path Attestation Result
+
+    Produces: (unfilled) Path Evidence 
+
+Verifier: A role performed by an entity that appraises the validity of filled Path Evidence about a set of Attesters and produces Path Attestation Results to be used by an Orchestrator.
+
+    Consumes: (filled) Path Evidence
+
+    Produces: Path Attestation Results
+
+# Conceptual Messages {#messages}
+
+The existing artifacts from RATS Architecture document {{RFC9344}} applies. New conceptual message(s) are defined here. 
+
+Path Request: A set of claims, describing the properties of a network path that a Relying Party requested.
+
+    Consumed By: Orchestrator
+
+    Produced By: Relying Party
+
+Path Attestation Result: The output generated by a Verifier, including information about a set of Attesters, where the Verifier vouches for the validity of the results. 
```

Comment:
The output generated by a Verifier, including information about individual or a set of Attesters, where the Verifier vouches for the validity of the results.